tools/check-shebang.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+FAILED=""
+FILES=0
+for F in $(find . -perm -u=x -type f | egrep -v "/vendor/|/gen/|/.git"); do
```

Comment:
It feels mildly suboptimal to grab all the executable files. Maybe rename the .pl above and limit the dragnet to .sh? This would give an option of requiring a shebang which (if I read it correctly) currently could be omitted.

Reply:
+1
Mark, please delete the .pl script.
I sometimes forgot add shebang at all, on Linux it somehow works.

Reply:
We've had the same issue with other types of scripts though. Commit 502ca6cde7 for example.

I'm not really sure why looking for executable files is suboptimal: the find(1) invocation in question returns 23 files.

Reply:
Fair enough.